COMPILING.md

```diff
@@ -309,3 +309,22 @@ If compiling with cmake:
    ```
    cmake --build build
    ```
+
+## Use BDDs for guards
+
+There are two implementation for symex guards. The default one uses the
+internal representation of expression. The other one uses BDDs and
+though experimental, it is expected to have better performances on
+difficult programs, in particular when used in conjunction with CUDD.
+
+To use the BDD implementation of guards, add the `BDD_GUARDS`
+compilation flag:
+  * If compiling with make:
+    ```
+    make -C src CXX_FLAGS="-DBDD_GUARDS"
```

Comment:
`CXXFLAGS` (no underscore) - makes me wonder: is this actually tested? And you will want to recommend something like `CXXFLAGS="-O2 -DBDD_GUARDS"` as otherwise the performance will be rather terrible...

Reply:
Corrected. I added a travis step with the flag, But it is using cmake. Is it worth adding a step with make and BDD_GUARDS?

Reply:
I think it's ok if you just test it at least once locally. I don't really expect random breakage of the Makefile-based builds with this flag. Alternatively, you could just add the flag to one of the existing Makefile-based build configurations.